src/App.tsx

```diff
@@ -17,18 +40,41 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                query={query}
+                filterBySelect={filterBySelect}
+                onSetQuery={setQuery}
+                onSetFilterBySelect={setFilterBySelect}
+              />
             </div>
 
-            <div className="block">
-              <Loader />
-              <TodoList />
-            </div>
+            {errorGetTodos || (
+              <span>
+                {errorGetTodos}
+              </span>
+            )}
+
+            {allTodos.length === 0 || errorGetTodos
```

Comment:
```suggestion
            {!allTodos.length || errorGetTodos
```

Create a variable for this condition